### PR TITLE
Docs - Upgrade and breaking changes for 0.13.2

### PR DESCRIPTION
**Pull request by pmbrull**
### Describe your changes :

- Breaking changes for 0.13.2
- Moved 0.12.3 changes to its own doc. To be removed in 1.0.

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Comment by pmbrull** [comment]
@TeddyCr could you please add the details on the `EntityName` changes?

**Comment by pmbrull** [comment]
Also @akash-jain-10 please validate the upgrade docs and see what versions/paths need to be updated 🙏

**Comment by pmbrull** [comment]
@akash-jain-10 already applied the upgrade docs changes. If you could please take a look

**Comment by TeddyCr** [comment]
> @TeddyCr could you please add the details on the `EntityName` changes?

Pushed the changes for `entityName` and `entityLink`

File: openmetadata-docs/content/overview/roadmap/index.md [hunk 1]
```diff
@@ -72,20 +72,12 @@ You can check the latest release [here](/overview/releases).
...
-  >
-    <li>We are super excited about this feature coming in 0.13.1</li>
-  </Tile>
 
 </TileContainer>
 
 
 
-## 1.0 Release - Feb 6th, 2023
+## 1.0 Release - Marc 14th, 2023
```
Comment (pmbrull) [inline openmetadata-docs/content/overview/roadmap/index.md:80]:
```suggestion
## 1.0 Release - March 14th, 2023
```

**Review by Vj-L — APPROVED** [review]

**Pull request merged**